src/extensions/registries/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./status-bar-registry"
 export * from "./kube-object-detail-registry";
 export * from "./kube-object-menu-registry";
 export * from "./cluster-feature-registry"
+export * from "./kube-object-status-registry"
```

Comment:
Isn't this exporting all `KubeObjectStatus` stuff? Is that intentional?

Reply:
I moved `KubeObjectStatus` types to `K8sApi`. Feels more natural to have them there.